shell.nix

```diff
@@ -33,6 +33,7 @@ mkShell {
     jq
     llvmPackages.libclang
     nixpkgs-fmt
+    nix
     openapi-generator-cli
```

Comment:
openapi-generator-cli still needed?

Reply:
Yeah but not our fork, just the upstream version for the python code